lib/std/sums.nim

```diff
@@ -6,12 +6,16 @@
 #    See the file "copying.txt", included in this
 #    distribution, for details about the copyright.
 
-## Fast sumation functions.
+## Fast summation functions.
 
+runnableExamples:
+  let data = [1, 2, 3, 4, 5, 6, 7, 8, 9]
```

Comment:
IIUC there's no point in using those procs for `SomeInteger`; examples should use `float`

Reply:
Should I just convert those numbers to floats or rather come up with an example where `sumKbn` and `sumPairs` are more accurate than the standard `sum` proc from the `math` module. For example, adopted from the tests:
```nim
import std/math

var data: seq[float]
for n in 1 .. 1000:
  data.add(1.0 / n.float)

doAssert sum(data) != 7.485470860550345
doAssert sumKbn(data) == 7.485470860550345
doAssert sumPairs(data) == 7.485470860550345
```

Reply:
ideal would be a summation (eg analytical function or other) with a known mathematical value where we know the answer (via a math formula) and where sum would be less accurate. But if not, something simpler works

Reply:
see https://github.com/nim-lang/Nim/pull/16797#issuecomment-765887851

Reply:
why don't you use almostEqual?

Reply:
The new example would fail with `almostEqual`, since that uses a smaller epsilon.